app/helpers/course_helper.rb

```diff
@@ -55,4 +55,8 @@ def course_funding(course)
       'Fee paying (no salary)'
     end
   end
+
+  def course_applications_open(course)
+    course.attributes[:applications_open_from]&.to_date&.strftime("%-d %B %Y")
```

Comment:
I wonder if `%-d %B %Y` can be put into something reusable like `:govuk_date` (see also https://github.com/alphagov/govuk_admin_template/blob/0dfd5cffe8ec6ffa2d29767531f1e64826773b7e/lib/govuk_admin_template/engine.rb#L16-L31)

Reply:
👍 at some point I don't see why not